draft-ietf-quic-qpack.md

```diff
@@ -619,14 +619,22 @@ QPACK defines two unidirectional stream types:
    It carries an unframed sequence of decoder instructions from decoder
    to encoder.
 
-<!-- s/exactly/no more than/  ? -->
 HTTP/3 endpoints contain a QPACK encoder and decoder. Each endpoint MUST
-initiate a single encoder stream and decoder stream. Receipt of a second
-instance of either stream type be MUST treated as a connection error of type
+initiate at most one encoder stream and one decoder stream. Receipt of a second
```

Comment:
> Each endpoint MUST initiate at most one encoder stream and one decoder stream.

This may be interpreted as "each endpoint MUST create an encoder and a decoder stream," which is contrary to this PR.  I suggest we rephrase this.

Reply:
Agrees to @dtikhonov. Maybe adding something like following immediately after could be sufficient. "Endpoints MAY avoid creating encoder streams if they are not going to be used. They MAY avoid or defer the creation of a decoder stream."

Reply:
Maybe swap the order to "one decoder stream and at most one encoder stream" to make it less likely to be misread as "at most one encoder stream and at most one decoder stream".

Reply:
Thanks. FWIW. We discussed in person tying up this change and getting the PR merged soon.

Reply:
> Maybe swap the order to "one decoder stream and at most one encoder stream" to make it less likely to be misread as "at most one encoder stream and at most one decoder stream".

Perhaps this manifests a difference of opinion, but that's exactly how I read this text, and what I think it should say.  An implementation could defer creating both the decoder stream (because the peer hasn't yet sent anything that needs acknowledgement) and the encoder stream (because it doesn't intend to send anything on the dynamic table).

Reply:
In that case, I propose the wording "at most one encoder stream and at most one decoder stream".  The current text is ambigous, as evidenced by Mike and I parsing it differently.

Reply:
Ok, updated to use 'at most' for both decoder and encoder.